pkg/parser/consts.go

```diff
@@ -0,0 +1,36 @@
+/*
+Portions Copyright (c) Microsoft Corporation.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package parser
+
+// cloud-init destination file references.
+const (
+	cseHelpersScriptFilepath             = "/opt/azure/containers/provision_source.sh"
```

Comment:
we will need to talk about these in depth, fine for now, but we will likely have a different version of the scripts, that adhere more to an updated cse_cmd.sh env vars.

Reply:
Sounds good.
For now, our getter just returns this const. We can return different file paths based on other conditions if needed.
``` 
func getCSEHelpersFilepath() string {
	return cseHelpersScriptFilepath
}
```